edge-modules/api-proxy-module/templates/nginx_default_config.conf

```diff
@@ -38,25 +38,42 @@ http {
             rewrite ^(.*)$ /storage$1 last;
         }
         #endif_tag ${BLOB_UPLOAD_ROUTE_ADDRESS} 
-        #if_tag !${BLOB_UPLOAD_ROUTE_ADDRESS}
+        #if_tag boolean_expression[!(${BLOB_UPLOAD_ROUTE_ADDRESS})]
         if ($http_x_ms_version)
         {
             rewrite ^(.*)$ /parent$1 last;
         }
-        #endif_tag ${BLOB_UPLOAD_ROUTE_ADDRESS}          
+        #endif_tag boolean_expression[!(${BLOB_UPLOAD_ROUTE_ADDRESS})]          
 
         #if_tag ${DOCKER_REQUEST_ROUTE_ADDRESS}
         if ($uri ~ ^/v2*.)
         {
             rewrite ^(.*)$ /registry$1 last;
         }
         #endif_tag ${DOCKER_REQUEST_ROUTE_ADDRESS}
-        #if_tag !${DOCKER_REQUEST_ROUTE_ADDRESS}
+        #if_tag ${CONNECTED_ACR_ROUTE_ADDRESS}
+        location /v2 {
+            resolver 127.0.0.11;
+            proxy_http_version 1.1;
+            proxy_pass         http://${CONNECTED_ACR_ROUTE_ADDRESS};
+            proxy_set_header   X-Forwarded-Host $http_host;
+            proxy_set_header   X-Forwarded-Proto $scheme;
+        }
+
+        location /acr {
+            resolver 127.0.0.11;
+            proxy_http_version 1.1;
+            proxy_pass         http://${CONNECTED_ACR_ROUTE_ADDRESS};
```

Comment:
@huanwu @jaysterp @savaradh do we need to set client_max_body_size to a bigger value for /acr requests?

Reply:
Yes, need add a line: client_max_body_size 1000G

Reply:
Fixe!